code/modules/reagents/chemistry/recipes/special.dm

```diff
@@ -281,7 +281,10 @@ GLOBAL_LIST_INIT(medicine_reagents, build_medicine_reagents())
 /datum/chemical_reaction/randomized/metalgen/GetPossibleReagents(kind)
 	switch(kind)
 		if(RNGCHEM_INPUT)
-			return GLOB.medicine_reagents
+			var/list/forbidden_reagents = (
```

Comment:
you didnt make a list? this just evaluates to forbidden_reagents = /datum/reagent/medicine/strange_reagent/instant
also make it a static list

Reply:
Probably more maintainable to just exclude non-synth reagents from recipe selection

Reply:
> Probably more maintainable to just exclude non-synth reagents from recipe selection

Good point

Reply:
> you didnt make a list? this just evaluates to forbidden_reagents = /datum/reagent/medicine/strange_reagent/instant also make it a static list

It was in case any other reagents needed exceptions, although none came to mind